src/testing/RegistrationTestApp.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from 'supertest';
```

Comment:
Can we move all the tests to `/src/__tests__` just to keep everything grouped an in the same place?